hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphTransaction.java

```diff
@@ -1620,6 +1608,23 @@ private void lockForUpdateProperty(SchemaLabel schemaLabel,
         }
     }
 
+    private void removeLeftIndexIfNeeded(Map<Id, HugeVertex> vertices) {
```

Comment:
can mock and test it? use Whitebox to set removeLeftIndexOnOverwrite=true

Reply:
add test, but seems not work